runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -308,6 +305,23 @@ else if (poisonVarp >= VENOM_VALUE_CUTOFF)
 
 			lastPvpVarb = pvpVarb;
 		}
+
+		if (lastImbuedHeartVarb != imbuedHeartCooldownVarb && config.showImbuedHeart())
+		{
+			if (imbuedHeartCooldownVarb == 0)
+			{
+				log.debug("The Imbued Heart has regained its magical power, clearing timer");
+				removeGameTimer(IMBUEDHEART);
+				imbuedHeartTimer = null;
+			}
+			else if (imbuedHeartTimer == null)
```

Comment:
```suggestion
			else
```

Reply:
See my comment above, this would recreate the timer every 60 game ticks. I opted to change this for a flag, let me know if that looks okay to you.